tap_gitlab/utils.py

```diff
@@ -29,7 +30,7 @@ def wrapper(*args, **kwargs):
                 timer = threading.Timer(every, semaphore.release)
                 timer.setDaemon(True)  # allows the timer to be canceled on exit
                 timer.start()
-                return result
+            return result
```

Comment:
I'm pretty sure this maintains the intent of this decorator while allowing exceptions to be passed along, but I could use another set of eyes on this.